.gitignore

```diff
@@ -16,3 +16,4 @@ dymosim
 *.bak-mo
 *.moe
 *.mol
+.DS_Store
```

Comment:
```suggestion
.DS_Store

```

Reply:
Same thing for the new row being added.

Reply:
See https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline GitHub even marks the missing newline with a red symbol